projects/systelab-preferences/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "systelab-preferences",
-  "version": "9.1.0",
+  "version": "14.0.0",
```

Comment:
I like the idea to adapt versioning to Angular versions, maybe we need to discuss this in next task force meeting to start doing the same in all applications.

Reply:
Probably is a good idea for production applications. The problem probably is that the version of the company application not only are dependent from Angular. For example, in Beacon we are using Java and Sprinboot

Reply:
I meant in Systelab libraries, to specify clearly in the versioning semantics that major version will match from now on Angular version